azure-devops/azext_devops/dev/team/_help.py

```diff
@@ -38,6 +38,35 @@ def load_team_help():
     short-summary: Manage security groups
     """
 
+    helps['devops security group create'] = """
+    type: command
+    short-summary: Create a new Azure DevOps group or materialize an existing AAD group.
+    long-summary:
+    examples:
+          - name: Create an Azure DevOps Group with name and description
+            text: |
+              az devops security group create --name 'Some group name' --description
+              'Something to describe this group' 
+              
+          - name: Materialize an existing AAD Group with its origin ID/AAD object ID
+            text: |
+              Get object ID of the existing AAD group 
+              az ad group show -g {Group Name}
+              az devops security group create --origin-id {Object ID}
+              Use this descriptor to add this group to other Azure DevOps Groups.
+
+          - name: Materialize an existing AAD Group with its origin ID/AAD object ID and also add it to an AzureDevOps group.
```

Comment:
This should be limited to "Add an existing AAD group to an Azure DevOps group" - remaining including saying it is done with origin ID etc are all implementation details. Focus should be on the scenario.

Reply:
"Add an existing AAD group to an Azure DevOps group"  This looks confusing when you compare it to 'membership add command' where we are supporting scenario of addition of group to other group.

Reply:
Removing any context related to 'materializing a group' , since its internal to implementation details